docs/02-app/02-api-reference/05-next-config-js/ppr.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Partial Prerendering (Experimental)
+nav_title: ppr (Experimental)
+description: Learn how to enable Partial Prerendering (experimental) in Next.js 14.
+---
+
+> Partial Prerendering is an **experimental** feature and subject to change.
+
+Partial Prerendering is an experimental feature that allows static portions of a route to be prerendered and served from the cache with dynamic holes streamed in, all in a single HTTP request.
+
+Incremental support is available in Next.js 15, allowing you to opt-in to Partial Prerendering for specific routes and layouts.
+
+To enable Partial Prerendering, set the experimental `ppr` flag `incremental`:
+
+```js filename="next.config.js"
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  experimental: {
+    ppr: 'incremental',
+  },
+}
+
+module.exports = nextConfig
+```
+
+## Options
+
+- `incremental`: Allowing you to incrementally opt-in layouts and pages, should be used with the [`experimental_ppr`](/docs/app/api-reference/file-conventions/route-segment-config#experimental_ppr) route config option.
+- `true`: Enables Partial Prerendering for all routes in your application.
+
+We recommend using `incremental` first. Once all the routes in your application have PPR enabled, it’ll be considered safe for you to set the `ppr` value to `true`, and enable it for the entire app and all future routes.
+
+> **Good to know:**
+>
+> - When using `incremental`, routes that don't have `experimental_ppr` will default to `false` and will not be prerendered using Partial Prerendering. You need to explicitly opt-in to Partial Prerendering for each route.
+> - Partial Prerendering does not yet apply to client-side navigations. We are actively working on this.
+> - Partial Prerendering is designed for the [Node.js runtime](/docs/app/building-your-application/rendering/edge-and-nodejs-runtimes) only. Using the subset of the Node.js runtime is not needed when you can instantly serve the static shell.
```

Comment:
I can't quite tell what the second sentence is supposed to mean.

(Maybe a Anthony-based problem...?)

Reply:
It's saying PPR rendering the static bits is quicker than the using the Edge runtime.

Reply:
Ah. More clear, then?

```suggestion
> - Partial Prerendering is designed for the [Node.js runtime](/docs/app/building-your-application/rendering/edge-and-nodejs-runtimes) only. Using the Edge Runtime is not needed when you can instantly serve the static shell.
```

Reply:
Love it!